src/modules/transaction/components/Overview/Overview.js

```diff
@@ -135,42 +115,51 @@ const formatDates = (date, period) => {
 };
 
 const formatDistributionByValues = (distributions) =>
-  moduleCommands.map((id) =>
-    (distributions[id] ? parseInt(distributions[id], 10) : 0));
+  moduleCommands.map((id) => (distributions[id] ? parseInt(distributions[id], 10) : 0));
 
-const Overview = ({ t, txStats }) => {
+const Overview = () => {
+  const { t } = useTranslation();
+  const [params, setParams] = useState({ limit: 7, interval: 'day' });
   const [activeTab, setActiveTab] = useState('week');
   const colorPalette = getColorPalette(useTheme());
-  const distributionByType = formatDistributionByValues(
-    txStats.data.distributionByType,
-  );
-  const distributionByAmount = normalizeNumberRange(
-    txStats.data.distributionByAmount,
-  );
-  const { txCountList, txVolumeList, txDateList } = txStats.data.timeline.reduce(
-    (acc, item) => ({
-      txCountList: [...acc.txCountList, item.transactionCount],
-      txDateList: [
-        ...acc.txDateList,
-        formatDates(item.date, activeTab).slice(0, 2),
-      ],
-      txVolumeList: [...acc.txVolumeList, fromRawLsk(item.volume)],
-    }),
-    {
-      txCountList: [],
-      txDateList: [],
-      txVolumeList: [],
-    },
-  );
+  // Fallback token for transaction statistics // @TODO: Add selector for active token when available in service
```

Comment:
@soroushm There's some discussion to be had as regards filtering transaction statistics by token since the data returned by the endpoint now has chainIDs as keys

Reply:
ok we can create an other issues while the API is working